### PR TITLE
OCLOMRS-32: A user should retrieve and search through all the concepts in a specific source

### PR DESCRIPTION
**Pull request by shakirandagire**
# JIRA TICKET NAME:
[OCLOMRS-32: A user should retrieve and search through all the concepts in a specific source](https://issues.openmrs.org/browse/OCLOMRS-32)

# Summary:
As a user, I should be able to view and search through all the concepts in a specific source.

### CODE REVIEW

File: src/components/dashboard/components/concepts/SpecificConceptList.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import Loader from '../../../Loader';
+
+const SpecificConceptList = ({ concepts, fetching }) => {
+  if (concepts.length >= 1) {
+    return (
+      <div className="text-center mt-3">
+        <table className="table table-bordered table-striped">
+          <thead className="header head">
+            <tr>
+              <th style={{ maxWidth: '5rem' }}>ID</th>
```
Comment (emasys) [inline src/components/dashboard/components/concepts/SpecificConceptList.jsx]: You should avoid using inline styles

File: src/components/dashboard/components/concepts/SpecificConceptTable.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    concept: {
+      id,
+      concept_class,
+      datatype,
+      display_name,
+    },
+  } = concept;
+  return (
+    <tr className="concept-table">
+      <td style={{ maxWidth: '5rem' }}>{id}</td>
```
Comment (emasys) [inline src/components/dashboard/components/concepts/SpecificConceptTable.jsx]: Don't use inline styles

File: src/App.js [hunk 3]
```diff
@@ -23,6 +24,7 @@ const App = () => (
           <Route exact path="/dashboard" component={Authenticate(SourceSearch)} />
           <Route exact path="/dashboard/concepts" component={Authenticate(ConceptSearch)} />
           <Route exact path="/dashboard/dictionaries" component={Authenticate(DictionaryDisplay)} />
+          <Route exact path="/dashboard/concepts/:organisation/:name" component={Authenticate(SpecificConcept)} />
```
Comment (mwaz) [inline src/App.js]: is it organisation or organization?

File: src/tests/Dashboard/container/SpecificConcept.test.jsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  it('should render with specific concepts', () => {
+    const props = {
+      fetchConceptsActionTypes: jest.fn(),
+      concepts: [concepts],
+      isFetching: true,
+    };
+    const params = {
+      match: {
+        params: {
+          organisation: 'owner',
```
Comment (mwaz) [inline src/tests/Dashboard/container/SpecificConcept.test.jsx]: still think its organization

File: src/App.js (hunk 3 above)
Comment (shakirandagire) [reply src/App.js]: Organisation and organization mean the same thing.
They are alternative spellings of the same word, both being used in British English and only one being used in American English.
1. Both words can be used in British English.
2. Organization is the only correct spelling in American English.

Comment (mwaz) [reply src/App.js]: my suggestion would be to go with what is currently in use the one with a 'Z', slight error but might be confusing for one who needs to write a function for the organizations module

Comment (shakirandagire) [reply src/App.js]: I now understand what you meant. Let me make the changes.

File: src/tests/Dashboard/container/SpecificConcept.test.jsx (hunk 4 above)
Comment (shakirandagire) [reply src/tests/Dashboard/container/SpecificConcept.test.jsx]: I have made the changes

File: src/App.js (hunk 3 above)
Comment (shakirandagire) [reply src/App.js]: I have made the changes

File: src/components/dashboard/components/concepts/SpecificConceptTable.jsx (hunk 2 above)
Comment (shakirandagire) [reply src/components/dashboard/components/concepts/SpecificConceptTable.jsx]: I have made the changes

File: src/components/dashboard/components/concepts/SpecificConceptList.jsx (hunk 1 above)
Comment (shakirandagire) [reply src/components/dashboard/components/concepts/SpecificConceptList.jsx]: I have made the changes

**Pull request merged**